addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -322,9 +334,13 @@ def _evaluate_res_ids(self):
 
         :return: a list of IDs (empty list in case of falsy strings)"""
         self.ensure_one()
+        if not self.res_ids and self.res_ids_by_model:
```

Comment:
Somehow res_ids should be computed on the "reference" field, if we allow to pick a record (I suppose it is the purpose)